### ISSUE TITLE
V compiler error

### ISSUE THREAD

**Issue by Iskander0** [MajorHard/vraylib#8]
When I try to compile my code with the v compiler, I get the following error:

``iskander@pc ~/path/ $ v main.v

/home/iskander/.vmodules/damienfamed75/vraylib/text.v:65:2: error: unexpected name `fontSize`, expecting `,`
   63 |     text string,
   64 |     rec Rectangle
   65 |     fontSize, spacing f32
      |     ~~~~~~~~
   66 |     wordWrap bool,
   67 |     tint Color,``

**Comment by Iskander0** [comment]
V version: 0.1.29 d88e8bb

**Comment by waozixyz** [comment]
yeah there are a lot of bugs to get it working with the new version of raylib and vlang, check my fork, it seems to work now
